src/main/java/hu/wurfel/refference/school/student/StudentService.java

```diff
@@ -27,26 +24,20 @@ public class StudentService extends StudentCrudService {
 	private final ClassCrudService classCrudService;
 	private final DiaryCrudService diaryCrudService;
 	private final MarkCrudService markCrudService;
-	private final DiaryService diaryService;
-	private final ClassService classService;
-	private final MarkService markService;
-	private final SubjectService subjectService;
-	private final TeacherService teacherService;
+	private final SubjectCrudService subjectCrudService;
+	private final TeacherCrudService teacherCrudService;
 	private List<Student> rContent;
 
-
-	public StudentService(StudentRepository studentRepository, ClassCrudService classCrudService, DiaryCrudService diaryCrudService, MarkCrudService markCrudService, DiaryService diaryService, ClassService classService, MarkService markService, SubjectService subjectService, TeacherService teacherService) {
+	public StudentService(StudentRepository studentRepository, ClassCrudService classCrudService, DiaryCrudService diaryCrudService, MarkCrudService markCrudService, SubjectCrudService subjectCrudService, TeacherCrudService teacherCrudService) {
 		super(studentRepository);
 		this.classCrudService = classCrudService;
 		this.diaryCrudService = diaryCrudService;
 		this.markCrudService = markCrudService;
-		this.diaryService = diaryService;
-		this.classService = classService;
-		this.markService = markService;
-		this.subjectService = subjectService;
-		this.teacherService = teacherService;
+		this.subjectCrudService = subjectCrudService;
+		this.teacherCrudService = teacherCrudService;
```

Comment:
Jahogy te azért csináltál külön crud serviceket, hogy a mindent IS használhasd a mindenhol IS!!!
Well, problem solved, yes. 😄 
Az ilyen fajta filterezéssel és annak ilyen szemléletű implementációjával az a legnagyobb gond, hogy ha kell még egy entitás és ahhoz service, valamint azt is bele akarod építeni ebben a mindent IS tudok keresni dologba, akkor azt 6 helyen kell implementálni. Ha jön a 8., azt már 7 helyen. Das ist probléma. 
Majd ha lesz merszed, akkor alkalomadtán kérdezz rá nálam, hogy mi az az interface alapú collection autowire. 😄

Reply:
Yepp, ez volt a cél, és azt éreztem hogy problémás lesz, de ennek még énis megpróbálok utánajárni, meg úgy alapból az injection-öknek, perpill jelenlegi tudással ezt tudtam kihozni.

Reply:
Erről majd beszéljünk face2face, aztán ott pontosabban el tudom mondani, hogy ezen a "szinten" kb hogy lenne célszerű.